src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImpl.java

```diff
@@ -58,9 +58,9 @@ public Notification validate(GenerateUrlRequest generateUrlRequest, String appli
 
         notification.addError(validateCourt(generateUrlRequest.getFilingPackage(), courtDetails, applicationCode));
 
-        if(isNewSubmission(generateUrlRequest.getFilingPackage())) {
-            notification.addError(validateParties(generateUrlRequest.getFilingPackage()));
-        } else {
+        notification.addError(validateParties(generateUrlRequest.getFilingPackage()));
```

Comment:
I think we need to keep the check `if(isNewSubmission(generateUrlRequest.getFilingPackage())) {`

the rule is if there it's not a new submission parties can be empty, or we might need to change validate parties

if there is no courtFile parties are required and needs to be validated
if there is a courtFile parties are optional, but if there is parties they should be validated